socketio/src/client/builder.rs

```diff
@@ -234,6 +234,12 @@ impl ClientBuilder {
     ///     .expect("connection failed");
     ///
     /// // use the socket
+    /// let json_payload = json!({"token": 123});
```

Comment:
If you use the `json!` macro here, please also add the import `use serde_json::json` as well.

Reply:
Done, thanks!